trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/Headers.php

```diff
@@ -92,132 +92,112 @@ public function setHeaders(array $headers) {
         }
       }
 
-      // With the header type already verified to be one of the valid types, 
-      // push the header into the designated type temporary context array.
-      $context_headers[ $header['type'] ][ $index ] = $header['name'];
-    }
-    
-    // Set each header type context array.
-    foreach($this->types as $type) {  
-      $this->context[ $this->context_key ][ $type ] = $context_headers[ $type ] ?? [];
+      // With the header type already verified to be one of the valid types,
+      // create a header entry where it contains only the name and type keys.
```

Comment:
Expanding the comment to make it clear why there is this extra step b/c I hesitated on it for longer then I should have.

```suggestion
      // create a header entry where it contains only the name and type keys.
      // This is needed because there is nothing stopping the input $headers
      // array from containing additional keys.
```